projects/echelon/index.js

```diff
@@ -29,6 +52,15 @@ module.exports = {
         sdk.util.sumSingleBalance(balances, coin, cash);
       });
 
+      const isolatedPairAddresses = await getIsolatedPairAddresses();
+      const pairs = await Promise.all(isolatedPairAddresses.map(pairAddress => getPair(pairAddress)));
+      pairs.forEach(({ collateral_token, total_collateral_amount, asset_token, total_supply_amount }) => {
+        // isolated pair collateral
+        sdk.util.sumSingleBalance(balances, collateral_token, total_collateral_amount);
+        // isolated pair supply
+        sdk.util.sumSingleBalance(balances, asset_token, total_supply_amount);
```

Comment:
you need to exclude the borrowed amount here

Reply:
@g1nt0ki addressed